pygazebo/pygazebo.cc

```diff
@@ -440,8 +462,8 @@ PYBIND11_MODULE(pygazebo, m) {
            py::arg("sensor_scope_name"))
       .def("get_joint_state", &Agent::GetJointState, py::arg("joint_name"))
       .def("get_link_pose", &Agent::GetLinkPose, py::arg("link_name"))
-      .def("get_collision_count",
-           &Agent::GetCollisionCount,
+      .def("get_collisions",
+           &Agent::GetCollisions,
```

Comment:
Add a help message to describe the function in .def()

Reply:
Can you also add a help message for this function?

Reply:
done